draft-ietf-mls-protocol.md

```diff
@@ -1289,6 +1289,7 @@ struct {
     opaque group_id<0..255>;
     uint64 epoch;
     Sender sender;
+    opaque authenticated_data<0..2^32-1>;
```

Comment:
The authenticated data is not part of the content of an `MLSPlaintext`, it only exists when `MLSCiphertext` is used

Reply:
I don't understand this comment.  `MLSPlaintext` does have an `authenticated_data` field, and the messages that are used here come from a Commit, which is encrypted, it would also use an `MLSCiphertext`.

Anyways, I had asked @bifurcation about this earlier, and he said that it was accidentally omitted and should be added, which is why I did this.  If it in fact should be omitted, then I can revert the change, but I think that we should either add the field, or else explain why it isn't there because every other field from the beginning of `MLSPlaintext` up to the `signature` field is included.

Reply:
You are absolutely right, apologies for the confusion. The two should be aligned of course.

Reply:
Discussed at 20210526 Interim. Will split out authenticated_data change to another PR so it can be merged. Editorial changes continue to be reviewed.